core/src/main/java/org/apache/carbondata/core/datamap/dev/DataMap.java

```diff
@@ -42,6 +42,17 @@
    */
   List<Blocklet> prune(FilterResolverIntf filterExp);
 
+  // TODO Move this method to Abstract class
```

Comment:
This TODO is not valid now, right?

Reply:
In FG implementation I already created Abstract classes , so when we merge it we can move there